### PR TITLE
AVA 🌠

### PR DESCRIPTION
**Pull request by floatdrop**
Migration from tape to [AVA](https://github.com/sindresorhus/ava).

- [x] Tests runs successfully
- [x] Fix `body in GET` test on NodeJS 0.10

### CODE REVIEW

**Comment by arthurvr** [comment]
Yippeee! Nice work.

**Comment by sindresorhus** [comment]
Yay!

> Use ES2015 in tests (maybe after sindresorhus/ava#50)

Nah, don't have to wait for that. Tests will only be a tiny bit slower to start.

File: test/test-arguments.js [hunk 1]
```diff
@@ -35,23 +35,23 @@ test('url argument is required', function (t) {
 		});
 });
 
-test('accepts url.parse object as first argument', function (t) {
+test('arguments - accepts url.parse object as first argument', function (t) {
 	got({hostname: s.host, port: s.port, path: '/test'}, function (err, data) {
 		t.error(err);
```
Comment (sindresorhus) [inline test/test-arguments.js]: `t.error` => `t.ifError`

File: test/test-arguments.js [hunk 2]
```diff
@@ -35,23 +35,23 @@ test('url argument is required', function (t) {
 		});
```
Comment (sindresorhus) [inline test/test-arguments.js]: The above test could use `t.regexTest`

**Comment by sindresorhus** [comment]
> Fix body in GET test on NodeJS 0.10

Maybe try running it as `test.serial`. Might be some kind of race issue.

**Comment by floatdrop** [comment]
Boom :boom:

**Comment by sindresorhus** [comment]
:fireworks:

**Pull request merged**